data/kubevirt-ipam-controller/001-cert-manager.yaml

```diff
@@ -0,0 +1,5839 @@
+{{ if not .IsOpenshift }}
```

Comment:
Cert-manager dependency should be documented but not installed as part of component installation, we should installe it at CI or optional to the user.

Reply:
cc @RamLavi 
we are going to remove cert-manager from this PR
and it will be required to be installed manually (needed only U/S) for users thats need the new component

if you have anything against please let me know

Reply:
@qinqon @maiqueb 
should we deploy the CRD U/S or also just doc it ?
CNAO deploys it only on U/S anyhow

Reply:
Your question is confusing.

You ask if you should deploy the CRD (I assume you're speaking about the IPAMClaims CRD), then say that CNAO only deploys it upstream. 

I don't follow what your question actually is.

Yes, CNAO should deploy the CRD upstream. And avoid doing it downstream.

Reply:
i meant if we should consider to also not deploy the CRD on U/S like we are going to do for cert-manager
and just documenting that it should be documented

> Yes, CNAO should deploy the CRD upstream. And avoid doing it downstream.

anyhow ok, this is the current state i will keep it

Reply:
This component requires that particular CRD, **and** requires certificates mounted on the controller pods, for the webhook to work. 

It does **not** require cert-manager FWIW. Any other certificate library that puts the certificates in that path will work - for instance, as we do downstream. 

Thus, the component requires the `IPAMClaim`s CRD, but not cert-manager. We should recommend it.